### PR TITLE
Add Swiftly installation instructions

### LINKED ISSUE #37: No instructions for installing Swiftly itself
There should be some instructions about how to install Swiftly itself.
A command like this will work as a quick-start:
```
curl -L https://raw.githubusercontent.com/swift-server/swiftly/main/install/swiftly-install.sh | bash
```

### PR DESCRIPTION
**Pull request by MahdiBM**
Resolves #37

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,6 +4,13 @@ swiftly is a CLI tool for installing, managing, and switching between [Swift](ht
 
 Ongoing maintenance and stewardship of this project is led by the [SSWG](https://www.swift.org/sswg/).
 
+### Installation
+
+Just Run:
```
Comment (adam-fowler) [inline README.md:9]: Maybe make this a little more formal
```suggestion
To download Swiftly and install Swift, run the following in your terminal, then follow the on-screen instructions.
```
I ripped that from the rust installation screen

Comment (MahdiBM) [reply README.md:9]: I also ripped that "Just Run" from rustlings README 😅

**Review by adam-fowler — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (ktoso) [reply README.md]: Yeah generally let's avoid "simply" "just" etc, these steps may not be super trivial to some newcomers :)

**Review by ktoso — APPROVED** [review]

**Comment by MahdiBM** [comment]
I don't know if we should mention installing `curl` or not 🤔.
It could be something like this:

### Installation

To download Swiftly and install Swift, first make sure you have `curl` installed:
```bash
apt update && apt install curl
```
Then install Swiftly, and follow the next instructions:
```bash
curl -L https://raw.githubusercontent.com/swift-server/swiftly/main/install/swiftly-install.sh | bash
```

**Comment by MahdiBM** [comment]
Oppsie, that was a misclick 😅

**Review by patrickfreed — APPROVED** [review]
Thanks for the contribution! Note that these instructions are bound to change a bit once the landing page has been created (#47), but this is good to have in the meantime.

**Pull request merged**